content/beginner/060_helm/helm_nginx/installnginx.md

```diff
@@ -53,13 +53,13 @@ To access NGINX from outside the cluster, follow the steps below:
         Watch the status with: 'kubectl get svc --namespace default -w mywebserver-nginx'
 
     export SERVICE_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].port}" services mywebserver-nginx)
-    export SERVICE_IP=$(kubectl get svc --namespace default mywebserver-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    export SERVICE_IP=$(kubectl get svc --namespace default mywebserver-nginx -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
     echo "http://${SERVICE_IP}:${SERVICE_PORT}"
 {{< /output >}}
 
 In order to review the underlying Kubernetes services, pods and deployments, run:
 ```sh
-kubectl get svc,po,deploy
+kubectl get svc,pod,deploy
```

Comment:
This command use the short name, let's leave it as is please.

Reply:
I know, I clearly stated that this change is
> in in order to improve clarity of the command and what does it do